README.md

```diff
@@ -68,152 +86,73 @@ It describes the instructions below, as well as shows you how to edit a script.
-watch its execution in a browser in real time.
-
-For an example of running a script in live execution mode, try:
+```sh
+# iOS https://app.waldo.com/applications/ios/sessions?versionId=wiki
+SESSION_ID=[SessionID] npm run ios -- --spec onboarding.ts
 
-```shell
-VERSION_ID=[VersionID] SHOW_SESSION=1 npm run wdio
+# Android https://app.waldo.com/applications/android/sessions?versionId=wiki
+SESSION_ID=[SessionID] npm run android -- --spec onboarding.ts
 ```
 
-This mode is very useful when you want to watch the current behavior of a script, but you do
-not intend to modify it.
-
-### Interactive execution
-
-In this last mode, your script interacts with an ongoing remote session that _remains alive_
-when execution reaches the end of the script.
-
-In order to run in this mode, you must first launch a session manually in Waldo by going to
-https://app.waldo.com/applications/ios/sessions, and then copy the session ID from the URL (they
-are prefixed with `sess-`, such as `sess-1234567890abcdef`).
+[live-exec]: https://github.com/waldoapp/wdio-service/blob/main/Readme.md#live-execution
 
-As long as this session remains alive, you can execute your script against it. For an example
-of running a script in interactive execution mode, try:
-
-```shell
-SESSION_ID=[SessionID] npm run wdio
-```
+### Using you own application
 
-_Note_: in this mode, it is not necessary to specify `VERSION_ID` since the remote session was already
-started with a specific app version.
+If you want to use your own application, you need to
+[upload a binary](https://app.waldo.com/applications/0/builds/upload) to Waldo first. Once this is done the version ID
+can be specified in the `'appium:app'` capability inside the `wdio.conf.js` file.
 
-This mode is very useful for creating a new script or editing an existing one, since it allows you
-to quickly relaunch your app over and over without waiting for session initialization. In addition,
-this mode also allows you to perform some actions manually on the session in the browser, as well
-as use the tree inspector to determine the best way to locate an element.
+More information can be found in the [Specifying the application binary/build to target][specifying] section of the
+service readme.
 
-## List of run parameters
+[specifying]: https://github.com/waldoapp/wdio-service/blob/main/Readme.md#specifying-the-application-binarybuild-to-target
 
-- `VERSION_ID` is the `id` of a specific build file of an app (hence the “app version”) you want to target with your test.
-  See the [upload section](#upload-your-app) of the documentation.
+### Integrating in your own project
 
-- `SHOW_SESSION` enables [live execution](#live-execution).
+If you already have [a WebDriverIO project][existing-wdio] or want to [start from scratch][from-scratch] you can
+follow the corresponding sections of the service readme.
 
-- `SESSION_ID` enables [interactive execution](#interactive-execution).
```

Comment:
Is `SHOW_SESSION` mentioned anywhere now?

Reply:
It's mentioned in https://www.npmjs.com/package/@waldoapp/wdio-service#live-execution page and documented in https://www.npmjs.com/package/@waldoapp/wdio-service#showsession

And also present in the code sample of the "Running the sample scripts" section ?